ansible/roles/ansible-horizon-plugin/tasks/collect_compress.yml

```diff
@@ -39,9 +39,33 @@
   shell: find / -type d -path /sys -prune -o -name "*local_settings.d" 2>/dev/null | grep -E 'openstack_dashboard|horizon'
   register: os_local_settings_path
 
+- name: Set file names
+  set_fact:
+    file_names: "{{ os_local_settings_path.stdout.split('\n')}}"
+  when: os_local_settings_path.stdout != ""
+
+- set_fact:
+    os_local_settings_path_length: "{{ file_names|length }}"
+  when: os_local_settings_path.stdout != ""
+
 - name: HORIZON_CONFIG
   shell: . /openstack/venvs/horizon*/bin/activate && echo "HORIZON_CONFIG['customization_module'] = 'dashboards.overrides'" > `echo $os_local_settings_path | cut -d ' ' -f1 |{ read trilio_setting_path ; echo $trilio_setting_path/_001_trilio_dashboard.py ; }`
```

Comment:
this path may not be correct

Reply:
now we are taking horizon virtual environment path from user.